pkg/controllers/securitypolicy_controller.go

```diff
@@ -53,6 +59,15 @@ func (r *SecurityPolicyReconciler) Reconcile(ctx context.Context, req ctrl.Reque
 		return ctrl.Result{}, client.IgnoreNotFound(err)
 	}
 
+	// Since securitypolicy service can only be activated from nsx 3.2.0 onwards,
+	// So need to check nsx version before starting securitypolicy Reconcile
+	if !r.Service.NSXClient.CheckNsxVersion() {
```

Comment:
ok

Reply:
@timdengyun  By the way, I think there is no need to check in every reconcile, how about cache a global variable, if it returns true and skip it in following reconciles.

Reply:
Thanks xie. In fact, we have a flag var versionSupported stored inside if client
So, when checkNsxVersion function is called, it will check this flag firstly, it will work as a global var check.

Reply:
ok